html/static/main.css

```diff
@@ -12,8 +12,7 @@ h3 { font-size: 16px; }
 pre {
   background-color: #eff0f1;
   padding: 5px;
-  white-space: pre-wrap;
-  overflow-wrap: break-word;
+  overflow: auto;
```

Comment:
Could we revert the CSS change - `overflow:auto` causes a horizontal scrollbar which we would rather avoid.